src/patients/diagnoses/Diagnoses.tsx

```diff
@@ -50,11 +54,18 @@ const Diagnoses = (props: Props) => {
         </div>
       </div>
       <br />
-      <DiagnosesList patientId={patient.id} />
+      <Switch>
```

Comment:
Given that this view will now be either viewing a particular Diagnosis or a table of diagnoses, should we add a back button in the same row as the "Add Diagnosis" button? Although the "Diagnoses" tab still exists in the parent component, it could be a little disorienting to have the table view shift to a particular diagnosis view.

Reply:
This is a really good point. I'll keep this PR as-is so that we're not adding too many additional features here, but I'm considering posting an issue to HR so that we can address this.